src/header/header.scss

```diff
@@ -190,181 +136,191 @@
         @include themed {
           background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
         }
-
-        &:not(:focus-visible):not(:hover) {
-          @include themed {
-            background-color: t(iui-color-background-1);
-          }
-        }
       }
     }
-  }
 
-  .iui-header-button-icon {
-    margin: 0 $iui-l;
-    @include themed {
-      fill: t(iui-icons-color-actionable);
-    }
-  }
+    .iui-slim {
+      .iui-header-brand-icon {
+        width: $iui-icons-default;
+        height: $iui-icons-default;
+        margin: 0 $iui-l + $iui-xs;
+      }
 
-  > .iui-label {
-    margin-right: $iui-sm;
-    font-size: $iui-font-size-leading;
-    @media (prefers-reduced-motion: no-preference) {
-      transition: font-size $iui-speed-fast ease-out;
+      .iui-header-brand-label {
+        font-size: $iui-font-size;
+      }
     }
   }
 }
 
-@mixin iui-header-button-focus {
-  &:focus {
-    box-shadow: none;
-    outline-offset: -1px;
-
-    @include themed {
-      outline: 1px solid t(iui-color-foreground-primary);
-    }
-  }
-
-  &:focus:not(:focus-visible) {
-    outline: 0;
-  }
-}
-
-@mixin iui-header-button {
+@mixin iui-header-breadcrumbs-button {
+  @include iui-button;
+  @include iui-button-borderless;
   @include iui-button-size(large, borderless);
```

Comment:
after all these includes there are so many rules, that its hard to follow.. And there are some which are not used or overriden.. But I guess it would take too long to check one by one and mb write separately. So lets keep them, for now..

Reply:
Big rewrite, most if not all `@includes` have been removed.